packages/fxa-auth-db-mysql/lib/db/schema/patch-112-113.sql

```diff
@@ -0,0 +1,10 @@
+-- -- create userCustomers table for storing fxa user to stripe customer relationship
+
+
+CREATE TABLE IF NOT EXISTS userCustomers (
```

Comment:
Last nit I could think of here is maybe `userCustomers` should be `accountCustomers` for consistency, but wouldn't block on that